### PR TITLE
Fix the recent update to credentials.yml

### PR DESCRIPTION
**Pull request by krutten**
Updated credentials to store the DB name used in Vault

### CODE REVIEW

File: jobs/mariadb-blacksmith-plans/templates/plans/standalone/init [hunk 1]
```diff
@@ -4,5 +4,6 @@ set -eu
 
 safe gen $CREDENTIALS/standalone/system password
 safe gen $CREDENTIALS/standalone/system username
+safe set $CREDENTIALS/standalone/system dbname="mysql"
```
Comment (jhunt) [inline jobs/mariadb-blacksmith-plans/templates/plans/standalone/init]: Please change this to `$CREDENTIALS/standalone/database:name`

File: jobs/mariadb-blacksmith-plans/templates/plans/standalone/manifest.yml [hunk 2]
```diff
@@ -4,7 +4,7 @@ meta:
     azs: z1
   admin_password: (( vault $CREDENTIALS "/standalone/system:password" ))
   admin_username: (( vault $CREDENTIALS "/standalone/system:username" ))
-  credential_name_field: "service_instance_db"
+  credential_name_field: (( vault $CREDENTIALS "/standalone/system:dbname" ))
```
Comment (jhunt) [inline jobs/mariadb-blacksmith-plans/templates/plans/standalone/manifest.yml]: What is a "credential name field"?

**Review by jhunt — CHANGES_REQUESTED** [review]

File: jobs/mariadb-blacksmith-plans/templates/plans/standalone/manifest.yml (hunk 2 above)
Comment (krutten) [reply jobs/mariadb-blacksmith-plans/templates/plans/standalone/manifest.yml]: A name that will be updated to something better

File: jobs/mariadb-blacksmith-plans/templates/plans/standalone/init (hunk 1 above)
Comment (krutten) [reply jobs/mariadb-blacksmith-plans/templates/plans/standalone/init]: Updated

File: jobs/mariadb-blacksmith-plans/templates/plans/standalone/manifest.yml (hunk 2 above)
Comment (krutten) [reply jobs/mariadb-blacksmith-plans/templates/plans/standalone/manifest.yml]: updated

**Comment by krutten** [comment]
This PR change fails on existing DB's (created before Init added the dbname set), will need to rethink handling existing db's

**Pull request closed without merging**